index.js

```diff
@@ -50,6 +54,20 @@ module.exports = {
           .catch(this._errorMessage.bind(this));
       },
 
+      _tagVersion: function(version) {
+        var tagVariable = this.readConfig('tagVariable');
+
+        if (!tagVariable) { return; }
+
+        var dir = this.readConfig('distDir');
+        var indexFilePath = path.join(dir, 'index.html');
+        var indexContent = fs.readFileSync(indexFilePath, { encoding: 'utf8' });
+
+        indexContent = indexContent.replace(tagVariable, version);
```

Comment:
where is this defined in the `index.html` file?

Reply:
Here: https://github.com/waltznetworks/web-ui/pull/945/files#diff-b9707fe88797285e93483456d1389b61R7

Reply:
cool :+1: